Makefile

```diff
@@ -1,64 +1,66 @@
-
-include dependencies.makefile
+.DEFAULT_GOAL := build
```

Comment:
Why not default to `all` and allow easy inclusion of other potential targets to simplify development (like running tests, cleaning env before build, etc.) ?

Reply:
I'm not sure there's an added value of an `all` at the moment. The way that cargo works, there's no added value in trying to build incremental steps on top of it.

I also don't use `make` for development. I do see the value of the Makefile in giving people new to the project a clear entrypoint, but aside from that it doesn't really do much except wrap cargo commands.

Reply:
ACK, makes sense :+1:

Reply:
I put something like this in all my Makefiles, occasionally tailored to the specific project, and use it as the default target so when you run `make` or `make help`, you get a list of all the targets with docs:

```makefile
IMAGE_NAME ?= project/image
VERSION ?= `cat VERSION`

.PHONY: help
help:
	@echo "$(IMAGE_NAME):$(VERSION)"
	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'

.PHONY: example_target
example_target: ## Some description of what this target does
```